src/components/Pages/HomeSections/videoSection/VideoSection.tsx

```diff
@@ -1,36 +1,165 @@
+                                diColor="#fff"
+                            />
+                        </div>
+                        <Image
+                            alt="videoPeopleImage"
+                            src={videoSectionPeopleImage}
+                            quality={100}
+                            className={styles.videoPeopleImage}
+                        />
+                    </div>
+                </>
+            ) : (
+                <>
+                    <div className={styles.textWrapper}>
+                        <h1 className={styles.header}>{currentLang["How does FreeDi work?"]}</h1>
+                        <p className={styles.underHeaderP}>
+                        {currentLang["Let us walk you through how our app works"]}
+                        </p>
+                    </div>
+                    <div
+                        className={styles.videoWrapper}
+                    >
+                        <iframe
+                            src={`https://www.youtube.com/embed/qLznfpgj1E4?wmode=transparent&autoplay=1`}
+                            title="YouTube video player"
+                            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                            loading="lazy"
+                            allowFullScreen
+                            className={
+                                isVideoPlaying
+                                    ? `${styles.videoPlayer} ${styles.videoImage}`
+                                    : styles.none
+                            }
+                        ></iframe>
+                        <Image
+                            alt="videoImage"
+                            src={videoSectionImage}
+                            quality={100}
+                            className={
+                                isVideoPlaying ? styles.none : styles.videoImage
+                            }
+                            onClick={() => handlePlayVideo()}
+                        />
+                        <p
+                            className={
+                                isVideoPlaying ? styles.none : styles.timerP
+                            }
+                        >
+                            {currentLang["explained in 90 second"]}
+                        </p>
+                        <div
+                            className={
+                                isVideoPlaying
+                                    ? styles.none
+                                    : styles.logoWrapper
+                            }
+                        >
+                            <FreediLogoIcon size={"1.459rem"} color={"#fff"} />
+                            <FreediIcon
+                                height="1rem"
+                                width="3.22rem"
+                                freeColor="#FFE601"
+                                diColor="#fff"
```

Comment:
Try to use variables for the colors.
Also, the rem doesn't have to be so specific 1.5rem is okay instead of 1.459rem

Reply:
const yellow = "#FFE601"
and then freeColor={yellow};
like this?

Reply:
No, I meant in global.scss

it would be best if you initiated a variable like this: --name-description: #ffe601

and then you can use it anywhere you want like this: { color: var(--name-description) }

Reply:
it is a tsx file, it is possible to use scss variables with tsx file?

Reply:
Yes this would work the same as giving it a specific color:
<FreediIcon
          height="1.9rem"
          width="4.1rem"
          freeColor="var(--notFound-title)"
          diColor="#fff"
  />

Reply:
done, ty didnt know i can do it :)

Reply:
We learn everyday :)